recipes/entt/3.x.x/patches/0002-3.12.1-fix-gcc-7.0-7.3.patch

```diff
@@ -0,0 +1,20 @@
+--- a/src/entt/meta/meta.hpp
```

Comment:
Just one comment for the record, we'll let the patch because it was previously applied but in general we prefer to not apply patches to source code.

Reply:
I agree; not sure how important this patch is, it seems to be so that EnTT works with GCC 7, which I assume doesn't have full/correct constexpr support. That said, EnTT itself asks for a fully C++17 compiler, which GCC 7 is supposed to be as far as I know. Not sure how relevant it is anymore or how many people still use it, but the reason why I included an updated patch as well is indeed because it was already previously applied.